docs/usage/framework_usage.rst

```diff
@@ -35,21 +35,23 @@ For that use case RestrictedPython provides the possibility to pass an own polic
 
 A policy is basically a special ``NodeTransformer`` that could be instantiated with three params for ``errors``, ``warnings`` and ``used_names``, it should be a subclass of RestrictedPython.RestrictingNodeTransformer.
 
-.. testcode:: own_policy
+.. code-block:: python
```

Comment:
same, keep ``testcode``
please also look at the ``[group]`` element of ``sphinx.ext.doctest`` the group name defines a context and all tests in this group context are evaluated together.

Reply:
done